node/runtime/src/lib.rs

```diff
@@ -237,8 +323,15 @@ impl grandpa::Trait for Runtime {
 	type Event = Event;
 }
 
+parameter_types! {
+	pub const WindowSize: BlockNumber = DEFAULT_WINDOW_SIZE as _;
```

Comment:
```suggestion
	pub const WindowSize: BlockNumber = DEFAULT_WINDOW_SIZE.into();
```

Reply:
any reason why the function is preferred?

Reply:
I guess because it works with non builtin types like Permill and rust should inline that anyway

Reply:
By using `as`, you will loose any protection by the compiler against unsupported conversions.